lib/R/VEuPathDB_General/utils.R

```diff
@@ -0,0 +1,36 @@
+## General helper functions
+
+
+  # Write to file - will need to be a helper function
+  # writeDT <- function(dt, pattern=NULL, verbose=c(TRUE, FALSE)) {
+  #   outFileName <- basename(tempfile(pattern = pattern, tmpdir = tempdir(), fileext = ".tab"))
+  #   data.table::fwrite(dt, outFileName)
+
+  #   plot.data::logWithTime(paste('New computation file written:', outFileName), verbose)
+  #   return(outFileName)
+  # }
+
+
+  # Write to json ()
```

Comment:
similar thought to writeDT.. think the idea is generally right that well need to do this, itll exist as a function somewhere, and the function will look roughly like this one. but being uncertain about details yet, and these being intended as general helpers i think id: 

1. remove references to computations/ metadata etc from the logging. make them as generic as possible
2. allow a directory/ path to be passed as an arg so we can store files wherever wed like to whatever name wed like. if nothing is passed, then maybe default to a tmp dir

Reply:
i guess the other option is to make the function names much more specific. either way.. just so long as they dont have generic names and compute-specific intent.